client/my-sites/domains/domain-management/list/list-header.jsx

```diff
@@ -64,8 +65,12 @@ class ListHeader extends React.PureComponent {
 					<InfoPopover iconSize={ 18 }>
 						{ translate(
 							'You can receive email using your custom domain by using email forwarding or by ' +
-								'purchasing a G Suite subscription. Note that email forwarding requires a plan ' +
-								'subscription.'
+								'purchasing Titan Mail or %(googleMailService)s subscription. Note that ' +
```

Comment:
I think we're missing a word:

> purchasing **a** Titan Mail or %(googleMailService)s subscription. Note that 

But in my opinion, the following would be even better:

> purchasing Titan Mail or %(googleMailService)s. Note that